.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
```

Comment:
Let's move this to `pyproject.toml`. See https://github.com/glass-dev/glass/blob/9bcdbe8e97c7548090bc5554cb7d2cc9300da30e/pyproject.toml#L88-L101